### PR TITLE
[ntfs2mimir] attach admins from file

### PR DESCRIPTION
**Pull request by pbench**
Ntfs2mimir needs data about the administrative regions to enrich the stops before putting them into elasticsearch.
The data about the administrative regions  was collected from elasticsearch. 
This collection method causes a non-negligible load on elasticsearch, resulting in various timeouts and errors in bragi.

This PR adds the option to collect the data about administrative region from a local cosmogony file (the same file used to build the "admin" index in cosmogony2mimir).

If a "cosmogony_file" is configured in ntfs2mimir config file, then this file will be used. 
If such configuration is not present, ntfs2mimir will gather administrative regions from elasticsearch, as before.

The same thing is done for the binaries osm2mimir, openaddresses2mimir and poi2mimir.

For more details: https://navitia.atlassian.net/browse/NAV-1339

### CODE REVIEW

**Review by woshilapin — APPROVED** [review]

**Review by azime — APPROVED** [review]

File: config/bano2mimir/default.toml [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
 update_templates = true
 
+# If a cosmogony_file is configured, ntfs2mimir will read the admins from this file.
+# If such a file is not configured, ntfs2mimir will read the admins from elasticsearch
+
+# cosmogony_file = "/home/pascal/mimirsbrunn/cosmogony-europe.jsonl.gz"
+
+# configuration for reading the cosmogony file
+french_id_retrocompatibility = true
+langs = [ "fr" ]
```
Comment (remi-dupre) [inline config/bano2mimir/default.toml:11]: Maybe I'm a bit late to the party and it's not very important, but wouldn't it be better to have this part in its own cosmogony-related subsection? (and same for other binaries)

```suggestion
  [cosmogony]
  # If a cosmogony_file is configured, ntfs2mimir will read the admins from this file.
  # If such a file is not configured, ntfs2mimir will read the admins from elasticsearch
  # cosmogony_file = "/home/pascal/mimirsbrunn/cosmogony-europe.jsonl.gz"

  # configuration for reading the cosmogony file
  french_id_retrocompatibility = true
  langs = [ "fr" ]
```

File: src/pois.rs [hunk 2]
```diff
@@ -116,21 +121,34 @@ pub async fn index_pois(
...
+                        .await
+                }
+                Err(err) => {
+                    warn!("administratives regions not found in es db. {:?}", err);
+                    return Err(Error::AdminRetrieval {
+                        details: err.to_string(),
+                    });
+                }
+            }
+        };
```
Comment (remi-dupre) [inline src/pois.rs:150]: :warning:  That may be a suggest for another PR since the issues extends to other parts of the settings.

This if/else is duplicated in several places, maybe grouping cosmogony settings is a good way to abstract things ot be able to call something like `cosmogony_settings.load_admins()?`.

File: config/bano2mimir/default.toml (hunk 1 above)
Comment (pbench) [reply config/bano2mimir/default.toml]: I've put everything in a [admins] subsection

File: src/pois.rs (hunk 2 above)
Comment (pbench) [reply src/pois.rs]: done

**Review by remi-dupre — APPROVED** [review]
